zrml/simple-disputes/src/lib.rs

```diff
@@ -302,14 +419,16 @@ mod pallet {
             }
 
             Self::deposit_event(Event::MarketDisputed(market_id, outcome));
-            Self::calculate_actual_weight(|_| 0, num_disputes as u32, max_disputes as u32)
+            calculate_actual_weight(|_| 0, num_disputes as u32, max_disputes as u32)
```

Comment:
I am confused about this return value. It doesn't seem to be used anywhere, in fact the whole `on_dispute` function is only called during the setup of benchmarks in the PM pallet, in fuzz and in tests. Will this return value ever be used anywhere? If it will, it must be changed because currently it always implies a weight correction to 0 weight when `num_disputes != max_disputes`

Reply:
Hum... Totally missed this one, thanks for spotting it.

`on_dispute` was a PM call, thus the reason of `calculate_actual_weight`.